neuron/src/app/Neuron/CLI/Types.hs

```diff
@@ -60,11 +61,16 @@ data SearchBy
   | SearchByContent
   deriving (Eq, Show)
 
+data OpenCommand = OpenCommand
+  { zettelId :: Maybe Text
```

Comment:
This should be `Maybe ZettelID`.

Reply:
Or even just `Route`. Which would allow you to open other pages, like the search page.

Reply:
We might have to change the CLI spec:

```
# opens zindex
neuron open
# opens a particular zettel
neuron open --id foobar
# opens the search page
neuron open --search
```